### PR TITLE
Vignette linking individual data methods and {cfr} package functions

### PR DESCRIPTION
**Pull request by adamkucharski**
This change adds a new vignette `estimate_from_individual_data.Rmd` that outlines the theory for CFR estimation for individual-level data and how the functions in CFR can be used to account for: 1) biases caused by different delays to outcome for fatal and non-fatal cases; and 2) estimation when only total deaths are known.

There are no changes to underlying functions.

- [ X] I have read the CONTRIBUTING guidelines
- [X ] A new item has been added to `NEWS.md`
- [X ] Tests for the changes have been added (for bug fixes / features)
- [X ] Docs have been added / updated (for bug fixes / features)
- [ X] Checks have been run locally and pass

### CODE REVIEW

File: vignettes/estimate_from_individual_data.Rmd
Comment (CarmenTamayo) [inline vignettes/estimate_from_individual_data.Rmd:262]: but doesn't this mean that the filtering method is not really recommended in this scenario at all, and that users should use cfr_static as id doesn't rely on reported recoveries?

File: vignettes/estimate_from_individual_data.Rmd
Comment (CarmenTamayo) [inline vignettes/estimate_from_individual_data.Rmd:301]:
```suggestion
df_cases$deaths[nrow(df_cases)] <- sum(df$deaths)
```

File: vignettes/estimate_from_individual_data.Rmd
Comment (CarmenTamayo) [inline vignettes/estimate_from_individual_data.Rmd:293]: so this is the equivalent of saying that we know the total number of deaths by the end of the outbreak? it's cool! I think maybe we should add this info to the vignette too as it might help visualise the concept for readers

File: vignettes/estimate_from_individual_data.Rmd
Comment (CarmenTamayo) [inline vignettes/estimate_from_individual_data.Rmd:321]:
```suggestion
$$
In other words, during a rapidly growing epidemic, we can only say that the CFR lies somewhere between the ratio of total deaths to total cases (as of the last known count) and 1 (the worst-case scenario).
```

**Review by CarmenTamayo — CHANGES_REQUESTED** [review]
I've ran the code to make sure everything was working smoothly and left a few comments for improvements/clarifications

File: vignettes/estimate_from_individual_data.Rmd
Comment (adamkucharski) [reply vignettes/estimate_from_individual_data.Rmd:95]: Good idea

File: vignettes/estimate_from_individual_data.Rmd
Comment (adamkucharski) [reply vignettes/estimate_from_individual_data.Rmd:35]: This is have, but will clarify

File: vignettes/estimate_from_individual_data.Rmd
Comment (adamkucharski) [reply vignettes/estimate_from_individual_data.Rmd:262]: Good point - have swapped 'methods' to 'conclusions'

File: vignettes/estimate_from_individual_data.Rmd
Comment (adamkucharski) [reply vignettes/estimate_from_individual_data.Rmd:293]: Yep - internally the functions don't actually require the death timseries for the static calculation

**Comment by adamkucharski** [comment]
Thanks for the comments @CarmenTamayo ! Have now made edits so ready to merge once approved.

**Review by CarmenTamayo — APPROVED** [review]

**Pull request merged**
